db/models.py

```diff
@@ -50,3 +55,67 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=datetime.now)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
The `Order` model uses `settings.AUTH_USER_MODEL` for the `user` ForeignKey, which is correct. Ensure that `AUTH_USER_MODEL` is set correctly in your Django settings if you are using a custom user model.